module/TaskManagement/src/TaskManagement/Controller/TransitionsController.php

```diff
@@ -7,24 +7,37 @@
 use TaskManagement\Service\TaskService;
 use TaskManagement\Task;
 use Zend\Validator\InArray;
+use Zend\Permissions\Acl\Acl;
+use Application\Entity\User;
 
 class TransitionsController extends HATEOASRestfulController
 {
 	protected static $resourceOptions = array ('POST');
-	protected static $collectionOptions= array ();
+	protected static $collectionOptions= array ('POST');
 	
 	/**
 	 * @var TaskService
 	 */
 	private $taskService;
-	
-	public function __construct(TaskService $taskService) {
+	/**
+	 *@var \DateInterval
+	 */
+	protected $intervalForCloseTasks;
+	/**
+	 *
+	 * @var Acl
+	 */
+	private $acl;
```

Comment:
la funzione isAllowed è disponibile attraverso un plugin del controller, quindi non devi passare attraverso il servizio $acl, puoi chiamare direttamente `$this->isAllowed()`. La dipendenza dal servizio $acl va quindi rimossa